frontend/src/stylesheets/layouts.scss

```
@@ -17,7 +17,7 @@ $transition_time: .15s;
 $nav_height: 60px;
 
 .ff-layout--box {
-    font-family: 'Heebo';
+    font-family: 'Heebo', system-ui;
```

```suggestion
    font-family: 'Heebo, system-ui, sans-serif';
```

while system-ui is present on my box, I think we should add known safe backup `sans-serif` to as a catch all